### PR TITLE
feature footer

### PR DESCRIPTION
**Pull request by codenuk**

### CODE REVIEW

**Review by Knight-H — CHANGES_REQUESTED** [review]
## Things to Improve
- [ ] Remove package.json in github tracking stage
- [x] In `footer/index.js` , `withRouter` is not used, why did you include?
- [x] In `footer/index.js`, WTF WHERE IS QR CODE? WHERE IS LINE, FB, EMAIL? WTF WHY IS IT IN SVG????
- [x] In `footer/index.js`, style name should be camelCase.
- [x] In `footer/index.js`, `styles.footer` should case name to something like `footerBar` or something for more specific. 
- [x] In `footer/index.js`, `<li>` `title_bold` should be `<b>` 


## Things to Note
- FooterComponent in App.js is new in every page??

**Comment by codenuk** [comment]
@Knight-H Hello i edit code in page footer plz check

File: src/components/footer/index.js [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
                     <li>ขั้นตอนการสั่ง</li>
                     <li>ดูสถานะการสั่งซื้อ</li>
                     <li>ติดต่อเรา</li>
                 </ul>
-
+                <ul className={styles.colQrCode}>
+                    <li><b>Scan Line QR code</b></li>
+                    <li><QrCode /></li>
+                </ul>
+                <ul className={styles.lisrContract}>
```
Comment (Knight-H) [inline src/components/footer/index.js]:
```suggestion
                <ul className={styles.listContact}>
```

File: src/components/footer/index.js [hunk 2]
```diff
@@ -1,26 +1,37 @@
...
-const FooterComponent = ({ history }) => {
+import { ReactComponent as BannerFooter } from './banner-footer.svg';
+import { ReactComponent as QrCode } from './qr-code.svg';
+
+const FooterComponent = () => {
     return (
         <footer>
-            <div className={styles.bg_footer}>
+            <div className={styles.bgFooter}>
+                <BannerFooter  />
```
Comment (Knight-H) [inline src/components/footer/index.js]: Why is this here this time? Last time it was imported by css isn't it?

File: src/components/footer/index.module.scss [hunk 3]
```diff
@@ -11,24 +11,111 @@
...
     };
+    .colQrCode {
+        flex-basis: 150px;
+        padding: 2vw 0 0 0vw;
+        svg {
+            margin-left: 20px;
+        }
+    }
+
+    .lisrContract {
```
Comment (Knight-H) [inline src/components/footer/index.module.scss]:
```suggestion
    .listContact {
```

File: src/components/footer/index.module.scss [hunk 4]
```diff
@@ -11,24 +11,111 @@
...
-.bg_footer {
-    width: 100vw;
-    height: 100%;
+.bgFooter {
     background-image: url('./bg-footer.svg');
     background-size: 100%;
     padding-bottom: 1%;
+    display: flex;
+    margin: 0 auto;
+    flex-direction: row;
```
Comment (Knight-H) [inline src/components/footer/index.module.scss]:
```suggestion
```

By default it's row

File: src/components/footer/index.module.scss [hunk 5]
```diff
@@ -11,24 +11,111 @@
...
+                display:flex;
+                align-items: center;
+                justify-content: center;
+                background-color: #FFFFFF;
+                height: 25px;
+                border-radius: 25px;
+                color: black;
+                padding: 0 10px;
+                margin-left: 5px;
+            }
```
Comment (Knight-H) [inline src/components/footer/index.module.scss:67]: WHY THE HELL IS THIS CLASS COPIED AGAIN AND AGAIN? 3 FUCKING TIMES?

File: src/components/footer/index.module.scss [hunk 6]
```diff
@@ -11,24 +11,111 @@
...
+        .squareLine {
+            display:flex;
+            align-items: center;
+            height: 30px;
+            background-color: #21B54D;
+            border-radius: 25px;
+            color: white;
+            padding: 3px 10px;
+            font-size: 14px;
+            position: relative;
```
Comment (Knight-H) [inline src/components/footer/index.module.scss]: `.squareLine` and `squareFace` is not so fucking different. the only thing different is the background-color?  Use class the way it should be. STOP REPEATING YOURSELF OVER AND OVER. CLEAN IT.

**Review by Knight-H — CHANGES_REQUESTED** [review]
Make sure your code is DRY before submitting for review.

**Comment by Knight-H** [comment]
Oh, and also try to spell English correctly

**Comment by codenuk** [comment]
i update 3 page na. 
   1) footer but i think i must edit this page to responsive to day.
   2) upload file page
   3) approve page => miss 2 componant because each of componant is P.Donut

**Pull request merged**